src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let price = days * 40;
+
+  if (days >= 3 && days <= 6) {
+    price = price - 20;
+  } else if (days >= 7) {
+    price = price - 50;
+  }
```

Comment:
The 'else if' block is unnecessary after an 'if' block that contains a 'return' statement. Simplify the control flow by using 'return' statements inside the 'if' blocks.